src/app-layout/utils/use-drawers.ts

```diff
@@ -19,6 +19,7 @@ interface ToolsProps {
   tools: React.ReactNode | undefined;
   onToolsToggle: (newOpen: boolean) => void;
   ariaLabels: AppLayoutProps.Labels | undefined;
+  disableDrawersMerge?: boolean;
```

Comment:
Optionally, remove special handling for tools-only version. New design does not support it and we plan to remove this condition gradually